src/main/java/com/conveyal/r5/analyst/cluster/AnalystWorker.java

```diff
@@ -759,6 +780,27 @@ public void finishPriorityTask(GenericClusterRequest clusterRequest, InputStream
     }
 
     /**
+     * Report to the broker that the task taskId could not be processed due to errors.
+     * The broker should then pass the errors back up to the client that enqueued that task.
+     * That objects are always the same type (TaskError) so the client knows what to expect.
+     */
+    public void reportTaskErrors(int taskId, List<TaskError> taskErrors) {
```

Comment:
I should add a status code parameter to this. The scenario application errors are usually the result of a bad request, but we're not sure randomly caught exceptions deserve a 400, and we may use this function elsewhere.